packages/jellyfish-api-core/__tests__/category/wallet.test.ts

```diff
@@ -632,3 +632,185 @@ describe('masternode', () => {
+      expect(utxos.length).toBe(1)
+      utxos.forEach(utxo => {
+        expect(utxo.address).toBe('mwsZw8nF7pKxWH8eoKL9tPxTpaFkz7QeLU')
+        expect(utxo.amount.isEqualTo(0.00001)).toBe(true)
+      })
+    })
+  })
+
+  it('should send multiple address using sendMany', async () => {
+    const amounts: Record<string, number> = { mwsZw8nF7pKxWH8eoKL9tPxTpaFkz7QeLU: 0.00001, mswsMVsyGMj1FzDMbbxw2QW3KvQAv2FKiy: 0.00002 }
+    return await waitForExpect(async () => {
+      const transactionId = await client.wallet.sendMany(amounts)
+      expect(typeof transactionId).toBe('string')
+
+      // generate one block
+      await container.generate(1)
+
+      // check the corresponding UTXOs
+      const utxos = await getMatchingUTXO(transactionId, 'mwsZw8nF7pKxWH8eoKL9tPxTpaFkz7QeLU')
+      // In this case the we should only have one matching utxo
+      expect(utxos.length).toBe(1)
+      utxos.forEach(utxo => {
+        expect(utxo.address).toBe('mwsZw8nF7pKxWH8eoKL9tPxTpaFkz7QeLU')
+        expect(utxo.amount.isEqualTo(0.00001)).toBe(true)
+      })
+
+      const utxos2 = await getMatchingUTXO(transactionId, 'mswsMVsyGMj1FzDMbbxw2QW3KvQAv2FKiy')
+      // In this case the we should only have one matching utxo
+      expect(utxos2.length).toBe(1)
+      utxos2.forEach(utxo => {
+        expect(utxo.address).toBe('mswsMVsyGMj1FzDMbbxw2QW3KvQAv2FKiy')
+        expect(utxo.amount.isEqualTo(0.00002)).toBe(true)
+      })
+    })
+  })
+
+  it('should sendMany with comment', async () => {
+    const amounts: Record<string, number> = { mwsZw8nF7pKxWH8eoKL9tPxTpaFkz7QeLU: 0.00001, mswsMVsyGMj1FzDMbbxw2QW3KvQAv2FKiy: 0.00002 }
+    return await waitForExpect(async () => {
+      const options: SendManyOptions = {
+        comment: 'test comment'
+      }
+      const transactionId = await client.wallet.sendMany(amounts, [], options)
+
+      expect(typeof transactionId).toBe('string')
+
+      // NOTE(sp): How to test the comment is there and where
+    })
+  })
+
+  it('should sendMany with replaceable', async () => {
+    const amounts: Record<string, number> = { mwsZw8nF7pKxWH8eoKL9tPxTpaFkz7QeLU: 0.00001, mswsMVsyGMj1FzDMbbxw2QW3KvQAv2FKiy: 0.00002 }
+    return await waitForExpect(async () => {
+      const options: SendManyOptions = {
+        replaceable: true
+      }
+      const transactionId = await client.wallet.sendMany(amounts, [], options)
+
+      expect(typeof transactionId).toBe('string')
+    })
+  })
+
+  it('should sendMany with confTarget', async () => {
+    const amounts: Record<string, number> = { mwsZw8nF7pKxWH8eoKL9tPxTpaFkz7QeLU: 0.00001, mswsMVsyGMj1FzDMbbxw2QW3KvQAv2FKiy: 0.00002 }
+    return await waitForExpect(async () => {
+      const options: SendManyOptions = {
+        confTarget: 60
+      }
+      const transactionId = await client.wallet.sendMany(amounts, [], options)
+
+      expect(typeof transactionId).toBe('string')
+
+      // NOTE(sp): How to test the effect of confTarget
```

Comment:
perhaps try `getrawtransaction` -> generate block -> `getblock` with minconf

Reply:
@surangap btw we use this structure for TODO comments

 https://github.com/DeFiCh/jellyfish/blob/main/CONTRIBUTING.md#todo-comments

Reply:
confTarget is likely fees testing, meaning calculate the fee that confirms in 60 blocks. But you need to create 16MB of transactions to test that. Event mainnet don't hit that cap, so you don't need to test that behavior.